lib/reporters/html.js

```diff
@@ -35,7 +35,7 @@ exports = module.exports = HTML;
 /**
  * Stats template.
  */
-
+var statsBar = '<h1 id="progress-bar"></h1>';
```

Comment:
How about we use `<div>` or `<progress>` tag since it is not heading.

Reply:
@outsideris changed from h1 to a div